python/dglke/models/mxnet/score_fun.py

```diff
@@ -60,6 +60,13 @@ def edge_func(self, edges):
         score = head + rel - tail
         return {'score': self.gamma - nd.norm(score, ord=self.dist_ord, axis=-1)}
 
+    def infer(self, head_emb, rel_emb, tail_emb):
```

Comment:
can we use edge_func to implement it?

Reply:
some infer func is different from score_func for some algorithms.